pkg/accounts/validate.go

```diff
@@ -67,10 +63,14 @@ func initAccountValidation(logger log.Logger, accounts Repository, validations v
 		w = route.Responder(logger, w, r)
 
 		vars := mux.Vars(r)
-		userID := moovhttp.GetUserID(r)
 		customerID := vars["customerID"]
 		accountID := vars["accountID"]
 
+		organization := route.GetOrganization(w, r)
+		if organization == "" {
+			return
```

Comment:
Is this an error? What http status code should it return?

Reply:
This already returns 400
 https://github.com/moov-io/customers/blob/master/pkg/route/organization.go#L19